flux/src/test/java/com/groupbyinc/util/UrlFunctionsBugTest.java

```diff
@@ -0,0 +1,118 @@
+package com.groupbyinc.util;
+
+import com.groupbyinc.api.model.Navigation;
+import com.groupbyinc.api.model.refinement.RefinementValue;
+import com.groupbyinc.api.tags.UrlFunctions;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.servlet.jsp.JspException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by ferron on 4/22/15.
+ */
+public class UrlFunctionsBugTest {
+
+    private final String DEFAULT_BEAUTIFIER = "default";
+    private final String REFINEMENT_REGEX = "~((?=[\\w]*[=:]))";
```

Comment:
This should be defined in the AbstractQuery class and referenced here.  Keep it DRY.